automation/repeated_test.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+SCRIPT_PATH="$(
+    cd "$(dirname "${BASH_SOURCE[0]}")/" || exit
+    echo "$(pwd)/"
+)"
+
+NUM_TESTS=${NUM_TESTS:-3}
+if [[ -z ${NEW_TESTS} ]]; then
+    NEW_TESTS=$("${SCRIPT_PATH}"/new_tests.sh)
```

Comment:
could `new_tests.sh` become a function in this script? is there a reason to keep them separate?

Reply:
Yes, easier testing.

Reply:
Done.